examples/plotting/filtering.py

```diff
@@ -1,3 +1,15 @@
+''' A map representation of unemployment rate in US using the `US_Sates Dataset`_.
+This example demonstrates using IndexFilter, ColorMapper and HoverTool with
+basic plot elements such as patches. When hovering over the points,
+the state and its umemployment rate is shown.
+.. bokeh-example-metadata::
```

Comment:
blank like for readability / consistency with existing examples
```suggestion
the state and its umemployment rate is shown.

.. bokeh-example-metadata::
```